src/pages/Home/Home.types.ts

```
@@ -0,0 +1,3 @@
+export interface HomeProps {
+  // define your props here
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)